### PR TITLE
Use che-aws cluster profile in eclipse-che, che-incubator, devworkspace-operator jobs

### PR DESCRIPTION
**Pull request by dmytro-ndp**
Change list:
- Use **che-aws** cluster profile in _eclipse-che/*_, _che-incubator/*_, _devfile/devworkspace-operator_ jobs.
- Renew OWNERS.
- Removed old jobs up to Eclipse Che 7.80.x branches.
- Run jobs on OCP **4.14** instead of 4.8, 4.11. 4.12.
- Remove _che-incubator/che-test-harness_ jobs, because target https://github.com/che-incubator/che-test-harness repo had been archived.
- Added "**chectl-devworkspace-happy-path**" test to "_che-incubator-chectl-main_" job config.

JIRA issue: https://issues.redhat.com/browse/CRW-5370

Related PR: https://github.com/openshift/ci-tools/pull/3872

### CODE REVIEW

**Comment by dmytro-ndp** [comment]
@akurinnoy , @svor, @tolusha, @amisevsk , @artaleks9: could you, please, review the PR?

**Review by artaleks9 — APPROVED** [review]

**Review by svor — APPROVED** [review]
che-plugin-registry part looks good to me

**Review by akurinnoy — APPROVED** [review]
LGTM

**Review by amisevsk — APPROVED** [review]

File: ci-operator/config/che-incubator/chectl/OWNERS [hunk 1]
```diff
@@ -1,9 +1,5 @@
 approvers:
-- rhopp
-- flacatus
 - abazko
```
Comment (tolusha) [inline ci-operator/config/che-incubator/chectl/OWNERS:2]: My github nickname is `tolusha`, could you replace please?

File: ci-operator/jobs/che-incubator/chectl/OWNERS [hunk 2]
```diff
@@ -1,9 +1,5 @@
 approvers:
-- rhopp
-- flacatus
-- tolusha
+- abazko
```
Comment (tolusha) [inline ci-operator/jobs/che-incubator/chectl/OWNERS:2]: My github nick name is `tolusha`, could you also add `svor` to the lists?

File: ci-operator/config/che-incubator/chectl/OWNERS [hunk 3]
```diff
@@ -1,9 +1,5 @@
 approvers:
-- rhopp
-- flacatus
```
[the remark this answers is not in the document]
Comment (dmytro-ndp) [reply ci-operator/config/che-incubator/chectl/OWNERS:3]: Done

File: ci-operator/jobs/che-incubator/chectl/OWNERS (hunk 2 above)
Comment (dmytro-ndp) [reply ci-operator/jobs/che-incubator/chectl/OWNERS]: Fixed

File: ci-operator/config/eclipse-che/che-operator/OWNERS [hunk 4]
```diff
@@ -1,9 +1,5 @@
 approvers:
-- rhopp
-- flacatus
 - tolusha
```
[the remark this answers is not in the document]
Comment (dmytro-ndp) [reply ci-operator/config/eclipse-che/che-operator/OWNERS:2]: Done

**Review by tolusha — APPROVED** [review]

**Comment by dmytro-ndp** [comment]
rehearsals-ack

**Pull request merged**